assets/javascripts/kitten/components/cards/crowdfunding-card.js

```diff
@@ -45,13 +45,15 @@ class CrowdfundingCardComponent extends Component {
 
     const Tag = this.props.href ? 'a' : 'div'
 
+    const titlesStyles = [titlesMinHeight && styles.titles]
+
     return (
       <StyleRoot>
         <Tag {...others} style={styles.card} onClick={this.removeCurrentFocus}>
           {this.renderImage()}
           {this.renderHeader()}
 
-          <Marger bottom="2" style={titlesMinHeight && styles.titles}>
+          <Marger bottom="2" style={titlesStyles}>
```

Comment:
J'ai l'impression que c'est potentiellement un breaking change dû à `styled-components` qui va nous contraindre de checker tous les `<marger … style={ … }>` des apps.

```
The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX. in div (created by Context.Consumer) in StyledComponent (created by marger__MargerDiv) in Marger (created by CrowdfundingCardComponent) in div (created by CrowdfundingCardComponent) in div (created by StyleRoot) in StyleRoot (created by CrowdfundingCardComponent) in CrowdfundingCardComponent (created by Card) in Card (created by ArticleCard) in ArticleCard in div (created by Story) in div (created by Story) in Story
```

Reply:
`styled-components` n'est pas fan du `false` dans l'attribut `style` :/

Reply:
Et `style={ null }` c'est valide ? Si oui `Marger` pourrait se protéger du breaking change en faisant un truc genre `<div style={ this.props.style || null }>`

Reply:
Yes. C'est propre ca ! Pas de doute de breaking change chelou. Thx 👍